tmva/pymva/src/MethodPyAdaBoost.cxx

```diff
@@ -38,12 +38,11 @@
 #include "TMVA/VariableTransformBase.h"
 #include "TMVA/Results.h"
 
-#include "TMath.h"
-#include "Riostream.h"
 #include "TMatrix.h"
 #include "TMatrixD.h"
 #include "TVectorD.h"
 
+#include <iostream>
```

Comment:
Why do you put `iostream` here? I don't see where we use it?

Reply:
Same for other files

Reply:
I see only calls to the logger, so the `iostream` include should be only in the header of the logger, I suppose?

Reply:
Yes, you are right. Actually I start writing my script with tmva, but then improve it.
I will remove this and few other includes and let you know when it ready